MAINTAINERS-example

```diff
@@ -1,33 +1,23 @@
-# This project, topsikrit, has top-secret awesome sauce.
-#
-# Btw, note that this comment was human-edited and will be preserved
-# even if you add/remove maintainers using the gem. Nifty!
-
-maintainers: {
-    version: 1
-    issues: "https://github.com/csprojects/topsikrit/issues"
-    # This is a very interesting comment. It will also be preserved.
-    roles: {
-        core: {
-            "ada": {},
-            "turing": {},
-        },
-        subsystem: {
-            "gracehopper": { subsystem: "ENIAC" },
-        },
+{
+  "version": 1,
+  "maintained": false,
+  "people": [
+    {
+      "github": "gracehopper",
+      "email": "admiral.grace.hopper@usnavy.gov",
+      "name": "Grace Hopper",
+      "comment": "Maintains ENIAC subsystem"
+    },
+    {
+      "github": "turing",
+      "email": "at@cambridge.edu",
+      "name": "Alan Turing",
+      "comment": "Passed his own test"
+    },
+    {
+      "github": "ada",
+      "email": "ada@example.com",
```

Comment:
It's in git commits anyway....

On Aug 26, 2016 6:15 PM, "Henrik Lindberg" <notifications@github.com> wrote:

> In MAINTAINERS-example
> <https://github.com/kylog/maintainers/pull/1#discussion_r76501507>:
>
> > +  "people": [
> > +    {
> > +      "github": "gracehopper",
> > +      "email": "admiral.grace.hopper@usnavy.gov",
> > +      "name": "Grace Hopper",
> > +      "comment": "Maintains ENIAC subsystem"
> > +    },
> > +    {
> > +      "github": "turing",
> > +      "email": "at@cambridge.edu",
> > +      "name": "Alan Turing",
> > +      "comment": "Passed his own test"
> > +    },
> > +    {
> > +      "github": "ada",
> > +      "email": "ada@example.com",
>
> not sure having email addresses in clear text is a good idea in public
> repositories
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/kylog/maintainers/pull/1/files/19e3dbb35375ef5f60edab13198fd7d4156b3797#r76501507>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAbMURMUrAGFEXWTwOZNZbmDYjw9Z0yks5qj3OsgaJpZM4JudIk>
> .
>